### DISCUSSION TITLE
Hel post

### DISCUSSION THREAD

**Discussion by subrosumon** [community/community#70605] [New to GitHub]
Hi 
I am new of this community, i nee this archive installer as below link, but i am not getting this installer, can you pls help me to provide?
 https://github.com/Manager-io/Manager.msi/releases/tag/20.9.89 

Thanks.

**Comment by queenofcorgis** [comment]
If you're asking how to use the piece of software that you found in a repository on GitHub, the best way to contact the maintainers of that software is to:

- Check the README for instructions on how to operate the software and pointers to documentation or troubleshooting info.
- Check the [SUPPORT file](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-support-resources-to-your-project), if one exists, for instructions on how to best contact the maintainers for support.
- Check the [CONTRIBUTING guide](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/setting-guidelines-for-repository-contributors), if one exists. Sometimes if there isn't a SUPPORT file, the CONTRIBUTING guide will give instructions on how to contact the maintainers for support.

All of these documents can be found, if they exist, in the repository where you found the software itself.

**Comment by Shariar-Hasan** [comment]
Hello,
did you tried this one?
![image](https://github.com/community/community/assets/75330767/9249fe2e-ca65-4283-8e91-b5c69ee37bd3)

i hope this one is what you are asking for.

**Reply by subrosumon** [reply]
Hi,
yes, I need this but version should be 20.9.98 and i did not find.
Can you pls help me to get the mentioned version?

**Reply by Shariar-Hasan** [reply]
well for your concern, this repository has its first release on this
![image](https://github.com/community/community/assets/75330767/675586dd-c955-4719-957c-2adef9ec831a)
so you can not get a vewrsion before it was created.
🙂

edit: in addition,
i just found its sibling repository has first release of that
![image](https://github.com/community/community/assets/75330767/5dc433f4-c69a-43b8-a569-01487d1b8baf)


so there is no release for the version you asked.
if you get your answer, can you mark my answer as answered?

**Reply by subrosumon** [reply]
yes, i saw that, 21.4.42 is the first, but before this version i required.

Thanks

**Reply by Shariar-Hasan** [reply]
ok, i got it
